BoxContentSDK/BoxContentSDK/BOXURLRequestSerialization.m

```diff
@@ -1393,7 +1393,7 @@ - (instancetype)initWithCoder:(NSCoder *)decoder {
         return nil;
     }
 
-    self.format = (NSPropertyListFormat)[[decoder decodeObjectOfClass:[NSNumber class] forKey:NSStringFromSelector(@selector(format))] unsignedIntegerValue];
+    self.format = (NSPropertyListFormat)[[decoder decodeObjectOfClass:[NSNumber class] forKey:NSStringFromSelector(@selector((format)))] unsignedIntegerValue];
```

Comment:
Why is this needed? It isn't needed for 'writeOptions' on the next line for example...